Makefile

```diff
@@ -397,6 +391,9 @@ up-detach: build ## Bring up local Grapl and detach to return control to tty
 		pulumi
 	echo "Pulumi complete"
 
+.PHONY: up-detach
+up-detach: up ## Bring up local Grapl and detach to return control to tty
```

Comment:
I'd just go ahead and delete this. There's a small enough number of engineers working on this that telling them "don't use `up-detach` anymore" is reasonable. Also, there's nothing here to indicate that this is a short-term alias; I think there would be a good chance that this ends up staying around for longer, potentially causing confusion.